extensions/sql-migration/package.json

```diff
@@ -34,6 +35,12 @@
         "category": "%migration-command-category%",
         "icon": "./images/migration.svg"
       },
+      {
+        "command": "sqlmigration.newsupportrequest",
```

Comment:
You should disable this in the commandPalette section (unless you actually want it to show up there, but if you do then you should have it only show up when it makes sense)

Reply:
Thank you, the same is needed for the feedback command